### PR TITLE
Minor tweak of Ryans localization.md PR

### PR DESCRIPTION
**Pull request by Rick-Anderson**
@ryanbrandenburg  If this is close enough to what you want, merge and edit the result. If it's too far off suggest changes or PR this PR.

### CODE REVIEW

File: aspnetcore/fundamentals/localization.md [hunk 1]
```diff
@@ -171,7 +171,7 @@ Resource files using `@inject IViewLocalizer` in Razor views follow a similar pa
 
 If you don't use the `ResourcesPath` option, the *.resx* file for a view would be located in the same folder as the view.
 
-While it is possible to create a 'default' resource (one which lacks a language identifier in the name), we do not recommend doing so because it leads to behavioral changes when moving between machines with different OS languages.
+We recommend that you do **not** create a 'default' resource file (A resource file that doesn't contain a language identifier in the name, such as *A.fr.resx*). A default resource file leads to behavioral changes when moving or deploying to machines with different OS languages.
```
Comment (ryanbrandenburg) [inline aspnetcore/fundamentals/localization.md]: You're giving an example of the opposite of what you describe with `A.fr.resx`. A cleaner description on the second sentence might be `...leads to behavioral changes when run on machines with...`

**Comment by Rick-Anderson** [comment]
@ryanbrandenburg  updated - thx.

**Comment by Rick-Anderson** [comment]
@DerAlbertCom please review

**Comment by DerAlbertCom** [comment]
@Rick-Anderson for me it's ok, made it at lot clearer. Also the state before this PR had helped by my problem. The the current Online Version was to confusing for my little brain ;)

But now it is really clear that if someone creates a default files things will go wrong.

**Pull request merged**